dhall/tests/Dhall/Test/QuickCheck.hs

```diff
@@ -457,18 +464,40 @@ instance Arbitrary URL where
 
         authority <- arbitrary
 
-        path <- arbitrary
+        let validPChar =
+                Test.QuickCheck.frequency
+#if MIN_VERSION_QuickCheck(0,5,8)
```

Comment:
These version numbers look more like `containers`! ;)

```suggestion
#if MIN_VERSION_QuickCheck(2,14,0)
```

Reply:
Whoops!  You're right 🙂